styleguide/source/_patterns/01-molecules/09-search-results/search-results-columns-external.twig

```diff
@@ -0,0 +1,23 @@
+{#
+ # @file search-results-columns-external.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<div class="layout layout_two_up-large">
+  <div class="layout_two_up-large-primary search-result">
+    <div class="search-result_columns">
+      {% include 'atoms-rule-horizontal' %}
+      <div class="search-result_label-wrapper">
+        {% include 'atoms-h6' with { 'class' : 'search-result_label', 'content' : 'AMA Wire' } %}
```

Comment:
might be able to reuse that label i was talking about here.

I also want to note for your reference that the h6 referenced in the designs is not the same as the h6 in the styleguide. This may require some h6 updating throughout these docs.

I think I linked it in a different PR, but the original h6/design reference sizes can be found in this document: https://drive.google.com/drive/folders/0B4-QX5V_ClpsdzhScW1IYWsza2c